graph/graph_unix.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package graph
+
+// allowBaseParentImage allows images to define a custom parent that is not
+// transported with push/pull but already included with the installation.
+// Only used in Windows.
+const allowBaseParentImage = false
```

Comment:
I don't like the `var` as I don't like it do be mutable. Would rather use `runtime.GOOS` then.